bin/stress-test/README.md

```diff
@@ -22,6 +22,7 @@ The endpoints that you can test are:
 - `sync_state`
 - `sync_notes`
 - `check_nullifiers_by_prefix`
+- `load_state`
 
 Each benchmark accepts options to control the number of iterations and concurrency level.
```

Comment:
I don't think concurrency applies to `load_store` - right? If so, I'd re-phrase this a bit.